src/main/webapp/js/common.es6

```diff
@@ -549,43 +413,147 @@ function sortByDiff(a, b) {
     return sortBase(a0, b0);
 }
 
+// http://stackoverflow.com/questions/7558182/sort-a-table-fast-by-its-first-column-with-javascript-or-jquery
 /**
- * To get point value from a formatted string
+ * Sorts a table based on certain column and comparator
  *
- * @param s
- *     A table cell (td tag) that contains the formatted string
- * @param ditchZero
- *     Whether 0% should be treated as lower than -90 or not
- * @returns
+ * @param oneOfTableCell
+ *     One of the table cell
+ * @param colIdx
+ *     The column index (1-based) as key for the sort
+ * @param ascending
+ *     if this is true, it will be ascending order, else it will be descending order
  */
-function getPointValue(s, ditchZero) {
-    let s0 = s;
-    const baseValue = 100;
+function sortTable(oneOfTableCell, colIdx, comp, ascending, row) {
+    // Get the table
+    let $table = $(oneOfTableCell);
 
-    if (s0.indexOf('/') !== -1) {
-        // magic expressions below as these cases will only be compared with
-        // case E +(-)X% (0 <= X <= 100)
-        if (s0.indexOf('S') !== -1) {
-            return 2 * baseValue + 1; // Case N/S (feedback contribution not sure)
+    if (!$table.is('table')) {
+        $table = $table.parents('table');
+    }
+
+    let columnType = 0;
```

Comment:
It looks even weirder than that... it's a `let`, so probably it's being changed somewhere further down.